modules/services/osm.js

```diff
@@ -480,7 +494,8 @@ export default {
         } else {
             var url = urlroot + path;
             var controller = new AbortController();
-            d3_xml(url, { signal: controller.signal })
+            var promiseXml = d3_xml(url, { signal: controller.signal, credentials: credentialReq() });
```

Comment:
> No need to assign to a separate variable here, although it doesn't hurt anything.

Not sure what you mean. Are u saying it could just be the following:
promiseXml = d3_xml(url, { signal: controller.signal, credentials: credentialReq() });

Reply:
Yep, that one could be `d3_xml(url, { signal: controller.signal, credentials: credentialReq() })` as the only necessary change. A separate variable is fine, too, if you prefer. I've been tending towards keeping the same shape of the code while modifying existing code, with the idea that then merge conflicts from upstream will be more straightforward to resolve.